app.js

```diff
@@ -1,14 +1,34 @@
+'use strict';
 
+/* External modules */
 
-var express = require('express'),
-  config = require('./config/config'),
-  glob = require('glob');
+var fs = require('fs');
+var express = require('express');
+var winston = require('winston');
+
+/* Library modules */
+
+var CampaignInfoStorage = require('./lib/storage/sqlite/campaign-info-storage');
+
+/* Controllers */
+
+var campaigninfoById = require('./controllers/campaigns').campaigninfoById;
+
+/* App variables */
 
 var app = express();
+var config = JSON.parse(fs.readFileSync('config.json'));
+var extern = { logger: winston };
 
```

Comment:
So in production, as you mentioned, we would want to setup a webserver in front which reverse proxies requests to our API on the backend. At least in the case of nginx, it's supposed to pass through any custom headers set by the backend application, but nginx (or whatever HTTP server serves as the proxy layer) always has the final say on what headers are sent in the response, so it can easily override or hide headers sent from the backend application.